packages/apollo/package-lock.json

```diff
@@ -1,325 +0,0 @@
-{
```

Comment:
Turned out that all these package locks within each package are not necessary. Probably a left over artifact from the initial setup.